tests/test_vector2_rotate.py

```diff
@@ -11,9 +11,14 @@
     (Vector2(math.pi, math.e), 67, Vector2(-1.27467, 3.95397))
 ]
 
+def isclose(x, y, epsilon = 6.5e-5):
+    d = (x - y) % 360
+    return (d < epsilon) or (d > 360 - epsilon)
+
 @pytest.mark.parametrize('input, degrees, expected', data)
 def test_multiple_rotations(input, degrees, expected):
     assert input.rotate(degrees) == expected
+    assert isclose(input.angle(expected), degrees)
```

Comment:
👍

Reply:
I had to write a custom `isclose` (rather than `math.isclose`) to deal with arithmetic modulo 360°, but I'm also pretty happy with it  :)